### PR TITLE
Novacustom ns5x/rel 1.0.0

### PR DESCRIPTION
**Pull request by mkopec**

### CODE REVIEW

**Comment by mkopec** [comment]
@miczyg1 please review

File: src/mainboard/clevo/tgl-u/mainboard.c [hunk 1]
```diff
@@ -30,6 +32,24 @@ static void mainboard_init(void *chip_info)
...
+	uint8_t data;
+
+	/*
+	 * NS5x comes with different touchpads from various vendors. Windows
+	 * gets confused when multiple touchpads are defined on the same I2C bus
+	 * in devicetree, so detect which touchpad variant is actually attached
+	 * to the mainboard and disable the other touchpads.
+	 */
+	for (child = i2c_dev->link_list->children; child; child = child->sibling)
+		if (child->path.i2c.device)
```
Comment (miczyg1) [inline src/mainboard/clevo/tgl-u/mainboard.c:47]:
```suggestion
		if (child && child->path.i2c.device)
```

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: src/mainboard/clevo/tgl-u/mainboard.c (hunk 1 above)
Comment (miczyg1) [reply src/mainboard/clevo/tgl-u/mainboard.c]: Ahh my bad, the for loop has the exit condition for child to be not null... You can revert the change

Comment (mkopec) [reply src/mainboard/clevo/tgl-u/mainboard.c:47]: Reverted

**Review by miczyg1 — APPROVED** [review]

**Pull request merged**